apps/explorer/src/types/assets.ts

```diff
@@ -44,6 +44,8 @@ export type NFTFile = {
   image?: string
 }
 
+export type AssetPriceResponse = { [tokenId: string]: { [currency: string]: number } }
```

Comment:
This could be added in the shared package, so that we can reuse it in our wallets!

Reply:
Let's deal with moving all the types in another PR?

Reply:
Cf. #134